pages/linux/kpackagetool5.md

```diff
@@ -0,0 +1,24 @@
+# kpackagetool5
+
+> KPackage Manager: Install, list, remove Plasma packages.
+> More information: https://techbase.kde.org/Development/Tutorials/Plasma5/QML2/GettingStarted#Kpackagetool5.
+
+- List all known package types that can be installed:
+
+`kpackagetool5 --list-types`
+
+- Install the package from a directory:
+
+`kpackagetool5 -t {{package_type}} -i {{path/to/directory}}`
```

Comment:
Is there a long form of any of these arguments? In tldr pages we generally prefer long-form options except in specific circumstances (e.g. `ls`).

Reply:
Done!